docs/source/transaction_family_specifications/track_and_trade_transaction_family.rst

```diff
@@ -0,0 +1,717 @@
+************************************************
+Track and Trade Transaction Family Specification
+************************************************
+
+Overview
+========
+
+The Track and Trade (T&T) transaction family allows users to track
+goods as they move through a supply chain. Records for goods include a
+history of ownership and custodianship, as well as histories for a
+variety of properties such a temperature and location. These
+properties are managed through a user-specifiable system of record
+types.
+
+
+State
+=====
+
+All T&T objects are serialized using Protocol Buffers before being
+stored in state. These objects include: Agents, Properties
+(accompanied by their auxiliary PropertyPage objects), Proposals,
+Records, and RecordTypes. As described in the Addressing section
+below, these objects are stored in separate sub-namespaces under the
+T&T namespace. To handle hash collisions, all objects are stored in
+lists within protobuf "Container" objects, the order of which is
+described below.
+
+
+Records
+-------
+
+Records represent the goods being tracked by Track and Trade. Almost
+every transaction references some Record.
+
+A Record contains a unique identifier, the name of a RecordType, and
+lists containing the history of its owners and custodians. It also
+contains a `final` flag indicating whether further updates can be made
```

Comment:
To specify in-line code you must use two backticks on each side like this:
```
``final``
```
Because RST is terrible.